components/availability/src/Availability.svelte

```diff
@@ -938,6 +944,19 @@
     days = [...days]; // re-render
     resetDragState();
   }
+
+  // Click action for un-draggable slot-list buttons (list view)
+  function toggleSlot(slot: SelectableSlot) {
+    if (slot.selectionStatus === SelectionStatus.SELECTED) {
+      slot.selectionStatus = SelectionStatus.UNSELECTED;
+    } else {
+      if (slotSelection.length < max_bookable_slots) {
```

Comment:
Just curious why this isn't `else if (slotSelection.length < max_bookable_slots) {...}`